parlai/scripts/display_data.py

```diff
@@ -22,8 +22,9 @@ def setup_args(parser=None):
     if parser is None:
         parser = ParlaiParser(True, True)
     # Get command line arguments
-    parser.add_argument('-ne', '--num-examples', type=int, default=10)
-    parser.add_argument('-mdl', '--max-display-len', type=int, default=1000)
+    parser.add_argument('-ne', '--num_examples', type=int, default=10)
```

Comment:
changing this flag just here could be confusing because we use the same name elsewhere but with '-' instead of '_' (see for instance scripts/display_model.py, scripts/eval_model.py, scripts/convert_data_to_parlai_format.py, etc). maybe we should change it everywhere or nowhere?

Reply:
yes i would like to have the "_" everywhere, less confusing in the code, but i don't want to do it in this PR -- so you want me to change it back for now, or not?